introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, my name is Ash and I'm ready to learn the ways of the Dojo!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?